packages/client/hmi-client/src/components/widgets/tera-input-field.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+	<div
+		class="input-container"
+		:class="{ error: errorMessage, disabled: disabled, focused: focused }"
+		@click="focusInput"
+	>
+		<label for="custom-input" class="input-label">{{ label }}</label>
+		<input
```

Comment:
What about the input type?

Reply:
Changing so that I use `useAttrs()`, that way we can bind an input prop we want without explicitly defining it.